src/test/scala/com/microsoft/ml/spark/vw/VerifyVowpalWabbitFeaturizer.scala

```diff
@@ -115,6 +115,8 @@ class VerifyVowpalWabbitFeaturizer extends TestBase with TransformerFuzzing[Vowp
 
     assert(v1.numNonzeros == 2)
 
+    val namespaceFeatures = VowpalWabbitMurmur.hash("in", 0)
+    // TODO
```

Comment:
remove TODO?

Reply:
I wasn't sure if you left it for a reason, will remove thanks!